cfep-21.md

```diff
@@ -0,0 +1,60 @@
+
+<table>
+<tr><td> Title </td><td> Conda Forge-Bioconda Transfer Policy </td>
```

Comment:
```suggestion
<tr><td> Title </td><td> conda-forge / Bioconda Transfer Policy </td>
```

We style conda-forge as such, lowercase, with dash. Same elsewhere. I added the slash to avoid the two dashes.